.eslintrc.js

```diff
@@ -57,8 +58,15 @@ module.exports = {
     "node/no-unpublished-require": "off",
     "node/no-unsupported-features/es-syntax": "off",
 
+    "notice/notice":[ "error", { mustMatch: "This Source Code Form is subject to the terms of the Mozilla Public", "templateFile": "copyright.txt" } ],
+
+    "eol-last": "warn",
+    "eqeqeq": "off", // TODO: "warn",
     "no-unused-vars": [ "error", { vars: "all", args: "none", ignoreRestSiblings: false } ],
     "no-var": "off", // TODO: "warn",
+    "no-warning-comments": "off", // TODO: "warn",
     "prefer-const": "off", // TODO: "warn",
+    "quotes": "off", // TODO: [ "error", "double" ],
+    "semi": [ "off", "always" ], // TODO: Set up prettier?
```

Comment:
Why do we need these?

Reply:
Not sure if we do, but i just noticed some inconsistency in the code.

If we do end up using Prettier, we can remove a bunch of the style-based ESLint rules in our config. But I don't have a lot of experience w/ Svelte to know how well it plays w/ Prettier so it might be more hassle than it's worth.

Reply:
Got it! Thanks for explaining. Do you mind keeping this PR exclusively for the headers, leaving out all the other changes, since they might require discussion with the rest of the team? :)